config.yaml

```diff
@@ -2,18 +2,22 @@
 # See LICENSE file for licensing details.
 
 options:
-  git-sync::repo:
+  git_sync_repo:
     type: string
     description: URL to repo to sync against
-  git-sync::branch:
+  git_sync_branch:
     type: string
     description: branch to sync
     default: main
-  git-sync::wait:
+  git_sync_wait:
     type: int
     description: git-sync wait interval between syncs (seconds).
     default: 60
-  prometheus_relpath:
+  prom_rules:
     type: string
     description: relative path in repo to prometheus rules
     default: ""
+  loki_rules:
+    type: string
+    description: relative path in repo to loki rules
+    default: ""
```

Comment:
Why no default? Please do add the default here, it is helpful for the admins to know how to shape new repos.

Reply:
See comment above for prom.